### PR TITLE
Package opam-depext.1.1.4

### PR DESCRIPTION
**Pull request by rjbou**
### `opam-depext.1.1.4`
install OS distribution packages
opam-depext is a simple program intended to facilitate the interaction between
OPAM packages and the host package management system. It can query OPAM for the
right external dependencies on a set of packages, depending on the host OS, and
call the OS's package manager in the appropriate way to install them.

---
* Homepage: https://github.com/ocaml/opam-depext
* Source repo: git+https://github.com/ocaml/opam-depext.git#2.0
* Bug tracker: https://github.com/ocaml/opam-depext/issues

---
:camel: Pull-request generated by opam-publish v2.0.3

### CODE REVIEW

File: packages/opam-depext/opam-depext.1.1.4/opam [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+]
+authors: [
+  "Louis Gesbert <louis.gesbert@ocamlpro.com>"
+  "Anil Madhavapeddy <anil@recoil.org>"
+]
+license: "LGPL-2.1 with OCaml-LGPL-linking-exception"
+homepage: "https://github.com/ocaml/opam-depext"
+bug-reports: "https://github.com/ocaml/opam-depext/issues"
+depends: ["ocaml"]
+available: opam-version >= "2.0.0~beta5"
```
Comment (kit-ty-kate) [inline packages/opam-depext/opam-depext.1.1.4/opam:14]: Should this be marked incompatible with opam 2.1~alpha? (not sure wish version brought back compatibility with it)

Comment (rjbou) [reply packages/opam-depext/opam-depext.1.1.4/opam:14]: Only the 2.1.0~alpha, it was fixed in [alpha2](https://github.com/ocaml/opam/releases/tag/2.1.0-alpha2)

**Comment by mseri** [comment]
All green. Can we merge?

**Comment by kit-ty-kate** [comment]
Yes! :tada: Thanks!

**Comment by rjbou** [comment]
Thanks!

**Pull request merged**